raiden/messages.py

```diff
@@ -99,6 +109,27 @@ def from_dict(data):
     return klass.from_dict(data)
 
 
+def message_from_sendevent(send_event, our_address):
+    if type(send_event) == SendLockedTransfer:
+        message = LockedTransfer.from_event(send_event)
+    elif type(send_event) == SendDirectTransfer:
+        message = DirectTransfer.from_event(send_event)
+    elif type(send_event) == SendRevealSecret:
+        message = RevealSecret.from_event(send_event)
+    elif type(send_event) == SendBalanceProof:
+        message = Secret.from_event(send_event)
```

Comment:
This agrees with the behaviour of `raiden_event_handler::handle_send_balanceproof()` where the proper thing to send is a secret, but just wanna check it again. This is correct, right? If you check the `if`-`elif` it's the only not same named message.

Reply:
Yes, it's the correct behavior, Secret and SendBalanceProof are just bad names